wdio.conf.js

```diff
@@ -94,7 +100,7 @@ exports.config = {
     baseUrl: 'https://bayut.com/',
     //
     // Default timeout for all waitFor* commands.
-    waitforTimeout: 99999999,
+    waitforTimeout: 1200000,
```

Comment:
`waitForTimeout` should not be greater than `mochaOpts.timeout`

Reply:
waitforTimeout updated